content/en/about/version-policy/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Versioning Policy
+description: Supported versions of various components and dependencies of Istio.
+weight: 15
+icon: cadence
+---
+
+## Istio Versioning
+
+Istio versions are expressed as **x**.**y**.**z**, where **x** is the major version, **y** is the minor version, and **z** is the patch version.
+
+Istio provides support for each minor release for 3 months after the next minor release. For example, Istio 1.0 support will be dropped 3 months after Istio 1.2 is released. See [Build & Release Cadence](/about/release-cadence/) for more details.
+
+## Component Version Skew
+
+Various components of Istio that interact with each other must be kept within supported version ranges. Below describes the required version skews. Where possible, we always recommend using the same version between all components.
+
+### Istioctl
+
+We currently recommend using `istioctl` within the same minor version as both the control plane and data plane.
+
+For example, `istioctl` version **1.0.1** could be used with control plane version **1.0.0** and data plane version **1.0.6**.
```

Comment:
This gets rather confusing very soon imo. How about something like “ istioctl works with same release of istio as well all the patch releases associated with it” or something along those lines.